src/store/cockpit/types.ts

```diff
@@ -18,3 +18,20 @@ export type Package = {
   version: string;
   release: string;
 };
+
+export type AWSWorkerConfig =
+  | {
+      bucket?: string | undefined;
+      region?: string | undefined;
```

Comment:
The region isn't actually used at all by the osbuild-worker, but it makes sense to save this info here for the on-prem version